bindings/python/iio.py

```diff
@@ -300,6 +301,7 @@ class ChannelType(Enum):
     _iiolib = "iio"
 
 _lib = _cdll("libiio.so.1", use_errno=True, use_last_error=True)
+_libc = _cdll(find_library("c"))
```

Comment:
This will fail in Windows. Need to use something like ```ctypes.util.find_msvcrt```

Probably need to fix the _cdll call above as well since that breaks Windows too.

Reply:
If Python bindings were broken on Windows before, and they are still broken on Windows after, then I'd argue it's not a problem of this commit and it can be fixed later in a subsequent PR. I'm kind of running against time here :)